src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -72,10 +76,13 @@ export default function FieldTemplate(props) {
   });
 
   const noWrapperContent =
-    !showFieldLabel &&
-    (schema.type === 'object' ||
-      schema.type === 'array' ||
-      (schema.type === 'boolean' && !uiSchema['ui:widget']));
+    (!showFieldLabel &&
+      (schema.type === 'object' ||
+        schema.type === 'array' ||
+        (schema.type === 'boolean' && !uiSchema['ui:widget']))) ||
+    (uiSchema &&
+      uiSchema['ui:options'] &&
+      uiSchema['ui:options'].noWrapperContent);
```

Comment:
Why does the existing functionality not work for you?

Reply:
Similar to my comment above, things were getting messed up when the date widget was within my custom field.